RELEASE.md

```diff
@@ -0,0 +1,20 @@
+Release type: minor
+
+Support "void" functions
+
+It is now possible to have a resolver that returns "None". Strawberry will automatically assign the new `Void` scalar in the schema
+and will always send `null` in the response
+
+## Exampe
+
+```python
+@strawberry.type
+class Mutation:
+    @strawberry.field
+    def do_something(self, arg: int) -> None:
+        return
+results in this schema:
+```grapqhl
+type Mutation {
+    doSomething(arg: Int!): Void
+}
```

Comment:
I think this file got messed up because of the suggestions I made 😊 can you fix it? I'm unable to push to your branch :)

Reply:
Whoops, I didn't notice :facepalm: